src/recipe/Baked.Recipe.Service.Application/CodingStyle/RichTransient/FindTargetFromInitializerConvention.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Baked.Business;
+using Baked.RestApi.Configuration;
+
+namespace Baked.CodingStyle.RichTransient;
+
+public class FindTargetFromInitializerConvention : IApiModelConvention<ActionModelContext>
+{
+    public void Apply(ActionModelContext context)
+    {
+        if (!context.Controller.MappedType.TryGetMembers(out var members)) { return; }
+        if (!members.Methods.Having<InitializerAttribute>().Any()) { return; }
+        if (context.Action.MappedMethod is null) { return; }
+        if (context.Action.MappedMethod.Has<InitializerAttribute>()) { return; }
```

Comment:
how does this not conflict with entities with public with :thinking:

Reply:
`LookupEntitiesByIdsConvention` overrides this conventions since this executes earlier